plugin-colorpicker/translations/colorpicker_ar.desktop

```diff
@@ -1,2 +0,0 @@
-Name[ar]=لاقط الألوان
```

Comment:
Why did you remove all those *.desktop translations?

Reply:
Are they still valid, @palinek? I changed the comment in `colorpicker.desktop.in`.

Reply:
Then, just remove the `Comment` key in localizations...

Reply:
@palinek, `.desktop` files in the `translations` directory should only contain the `Name[xx]` and `Comment[xx]` keys, shouldn't they? Most of them have the whole content of `colorpicker.desktop.in` plus the localized keys. It is to fix them if it is the case before I update the pull request.

Reply:
> should only contain the Name[xx] and Comment[xx] keys, shouldn't they? 

Yes. But during the build process everything except `Key[lang]` is discarded...

So you're right, you can discard contents copied from original .desktop file.